cluster/gce/util.sh

```diff
@@ -1852,9 +1873,11 @@ function make-gcloud-network-argument() {
 }
 
 # $1: version (required)
+# $2: Prefix for the template name, i.e. NODE_INSTANCE_PREFIX or
+#     WINDOWS_NODE_INSTANCE_PREFIX.
 function get-template-name-from-version() {
   # trim template name to pass gce name validation
-  echo "${NODE_INSTANCE_PREFIX}-template-${1}" | cut -c 1-63 | sed 's/[\.\+]/-/g;s/-*$//g'
+  echo "${2}-template-${1}" | cut -c 1-63 | sed 's/[\.\+]/-/g;s/-*$//g'
```

Comment:
Consider assigning the indexed arguments to local variables with human-readable names, in addition to the comment.

Reply:
Done.